scripts/auto-stop-idle/autostop.py

```diff
@@ -98,9 +98,11 @@ def get_notebook_name():
                         idle = False
                 else:
                     idle = False
+                    print('Notebook idle state set ', idle, ' due to kernel connections.')
```

Comment:
Suggest to use `print(f'Notebook idle state set as {}, because no kernel has been detected.', idle)`

Reply:
updated in new commit